pkg/storage/ec/client.go

```diff
@@ -199,11 +199,11 @@ func (ec *ecClient) Repair(ctx context.Context, limits []*pb.AddressedOrderLimit
 	successfulHashes = make([]*pb.PieceHash, len(limits))
 	var successfulCount int32
 
-	// how many nodes must be repaired to reach the success threshold: o - (n - r)
-	optimalCount := rs.OptimalThreshold() - (rs.TotalCount() - nonNilCount(limits))
-
-	ec.log.Sugar().Infof("Starting a timer for %s for repairing %s to %d nodes to reach the success threshold (%d nodes)...",
-		timeout, path, optimalCount, rs.OptimalThreshold())
+	{ // log num of pieces to be repaired to reach the success threshold
+		optimalCount := rs.OptimalThreshold() - nonNilCount(limits)
```

Comment:
Why this formula changed?

Reply:
Fro the reason commented in the PR message

> Log message was reporting the invalid number of pieces to be repaired for reaching the optimal/success threshold.

As far as I could understand the previous calculation was wrong, but I may be wrong, so please double check if this change make sense and the previous calculation was not correct.

Reply:
hmm isn't that we are trying to upload all `non nil limits` so we should display in log just `nonNilCount(limits)`.

Reply:
oh, yes you're right here, because I'm working in https://github.com/storj/storj/pull/2423 and I found this bug in that branch but I wanted to have a separated PR for this fix, I did the change needed by the changes of that PR.

So, yes, here is `nonNilCount(limits)` and I will do this change in the branch https://github.com/storj/storj/pull/2423

Reply:
Done https://github.com/storj/storj/pull/2454/commits/1ef9e2a9e8bcc6ea7d92235e81c9c22f23255304

I also rearranged some vars, around that log message, closer where they are used.